lib/matplotlib/backend_bases.py

```diff
@@ -1580,6 +1580,13 @@ class FigureCanvasBase:
     # interactive framework is required, or None otherwise.
     required_interactive_framework = None
 
+    # The manager class instantiated by new_manager.
+    # (This is defined as a classproperty because the manager class is
+    # currently defined *after* the canvas class, but one could also assign
+    # ``FigureCanvasBase.manager_class = FigureManagerBase``
+    # after defining both classes.)
+    manager_class = _api.classproperty(lambda cls: FigureManagerBase)
```

Comment:
Can the manager class be private?

Reply:
This is intended to be a public customization point for third-party backends, and should therefore be public, IMO.

Reply:
Technically, we'd want what other languages descripbe as *protected*: Accessible for subclasses, but not for users. I think this is still best translated to python by adding a leading underscore.

Reply:
Unfortunately, all we have is whether to put an underscore or not, and I would rather have that mean "do we allow ourselves to change this API without deprecation or not"; in this specific case the answer is clearly "not" as I think this party backends should indeed use it. (It's also not the end of the world if end users access this attribute, I think?)

Reply:
Fair.